### ISSUE TITLE
[thepiratebay] Exception (thepiratebay): Got redirected to another domain. Try changing the indexer URL to https://www.pirate-bay.net/.: Parse error (Test)

### ISSUE THREAD

**Issue by Bitpusher68** [Jackett/Jackett#6656]
An error occured while testing this indexer
Exception (thepiratebay): Got redirected to another domain. Try changing the indexer URL to https://www.pirate-bay.net/.: Parse error

url entered was https://thepiratebays3.com/ which is working in my browser.

**Comment by garfield69** [comment]
this is because the proxy https://thepiratebays3.com/ is not a true mirror of the official site https://thepiratebay.org/

when you do a search on TPB without keywords, the indexer fetches the _recent_ page.
however, https://thepiratebays3.com/recent redirects to https://www.pirate-bay.net/recent

And https://www.pirate-bay.net/ is not an official mirror either, the html it presents is not in the same layout as the official TPB site, so the Jackett indexer cannot process it.
so setting the site link to https://www.pirate-bay.net/ will not work either.

basically, you cannot use the unofficial proxies you found with Jackett's TPB indexer.
